app/liquid/views/partials/_form.liquid

```diff
@@ -10,6 +10,8 @@
         {% case field.data_type %}
         {% when 'text' or 'postal' %}
           <input type="text" name="{{field.name}}" class="form-control form__content" id="" placeholder="{{ field.label }}" maxlength="249" />
+        {% when 'hidden' %}
+          <input type="hidden" name="{{field.name}}" class="form-control form__content" id="" placeholder="{{ field.label }}">
```

Comment:
I know this is merged but looking on staging, I can't understand as a user how to set the value for a hidden form field. It still asks for placeholder text, which is odd given it's hidden, and doesn't ask for a value. Can you explain this? In general, I think we might need to think about how we can document stuff like this for users that can't just be done in a tooltip, as right now this is fairly opaque.